packages/cactus-plugin-ledger-connector-ethereum/src/main/typescript/plugin-ledger-connector-ethereum.ts

```diff
@@ -168,6 +172,9 @@ export class PluginLedgerConnectorEthereum
     app: Express,
     wsApi: SocketIoServer,
   ): Promise<IWebServiceEndpoint[]> {
+    // Add custom replacer to handle bigint responses correctly
+    app.set("json replacer", stringifyBigIntReplacer);
```

Comment:
@outSH This is a setting that's global to all plugins that are loaded into the API server so I would rather you put this into the `api-server.ts` file itself so that it's the single source of truth for global settings affecting all plugins. Otherwise people will have a hard time figuring out in production why their request bodies are  parsed/generated differently from the standard JSON if they don't realize that the plugins have access to the `Express` object of the API server (which we should somehow prevent anyway TBH)

Reply:
@petermetz Good idea, didn't think of it! I've moved this to api-server, please check if that's OK

Reply:
> @petermetz Good idea, didn't think of it! I've moved this to api-server, please check if that's OK

@outSH Yup, looks good to me, thank you!